packages/@secretlint/secretlint-rule-sendgrid/src/index.ts

```diff
@@ -0,0 +1,80 @@
+import {
+    SecretLintRuleContext,
+    SecretLintRuleCreator,
+    SecretLintRuleMessageTranslate,
+    SecretLintSourceCode,
+} from "@secretlint/types";
+import { matchPatterns } from "@textlint/regexp-string-matcher";
+
+require("string.prototype.matchall").shim();
+
+export const messages = {
+    SENDGRID_KEY: {
+        en: (props: { KEY: string }) => `found Sendgrid api key: ${props.KEY}`,
+        ja: (props: { KEY: string }) => `Sendgrid APIキーが見つかりました： ${props.KEY}`, // TODO: Needs review.
```

Comment:
Looks like good!
You can just remove todo.

```suggestion
        ja: (props: { KEY: string }) => `Sendgrid APIキーが見つかりました： ${props.KEY}`,
```